History/js/new.js

```diff
@@ -249,6 +246,37 @@ var callbacks = {
       // Clear the old iframe and insert the new one
       $(".privly_iframe").empty();
       $(".privly_iframe").append(iFrame);
+    }
+
+    $('button#prev_preview').on('click',function(){
+      //finds the previous button with preview_link class and passes its reference.
+      //more info : http://jsfiddle.net/5QdgB/
```

Comment:
But, that's because user is trying to go beyond the number of messages that are there. What should be printed in that case, then? Or, should we just roll back to the first message after the user clicks 'Next' on the last message?